tools/envoy_headersplit/README.md

```diff
@@ -0,0 +1,13 @@
+# Envoy Header Split
+Tool for spliting monolith header files in envoy to speed up compilation
```

Comment:
Spelling/grammar suggestion: "Tool for splitting monolithic header files in Envoy to improve compilation time"

Throughout this change, "monolith" is a noun, and "monolithic" is an adjective. We want "monolithic" in most places to describe the header file.

Reply:
fixed